core/frontend/src/menus.ts

```diff
@@ -0,0 +1,134 @@
+const menus = [
+  {
+    title: 'Main',
+    icon: 'mdi-home',
+    route: '/',
+  },
+  {
+    title: 'Vehicle',
+    icon: 'mdi-submarine',
+    submenus: [
+      {
+        title: 'General',
+        icon: 'mdi-image-filter-center-focus-strong',
+        route: '/autopilot/general',
+        advanced: false,
+        text: 'General Autopilot settings, allow you to start/stop Ardupilot if using Navigator or SITL, and switch'
+              + ' boards if more than one is present',
+      },
+      {
+        title: 'Firmware',
+        icon: 'mdi-chip',
+        route: '/autopilot/firmware',
+        advanced: false,
+        text: 'Used to download and flash new firmware to your board',
+      },
+      {
+        title: 'Log Browser',
+        icon: 'mdi-file-multiple',
+        route: '/autopilot/logs',
+        advanced: false,
+        text: 'Allow browsing the Telemetry (.tlog) and Binary (.bin) logs generated by your vehicle. Bin logs are'
+              + ' currently only supported for Navigator boards',
+      },
+      {
+        title: 'Endpoints',
+        icon: 'mdi-arrow-decision',
+        route: '/autopilot/endpoints',
+        advanced: true,
+        text: 'Manage Mavlink endpoints for internal/external systems. Used this if you need to connect additional'
+              + ' mavlink systems to your vehicle',
+      },
+      {
+        title: 'Video',
+        icon: 'mdi-video-vintage',
+        route: '/autopilot/video-manager',
+        advanced: false,
+        text: 'Manage your video devices and video streams',
+      },
+    ],
+  },
+  {
+    title: 'Tools',
+    icon: 'mdi-hammer-screwdriver',
+    submenus: [
+      {
+        title: 'Available Services',
+        icon: 'mdi-account-hard-hat',
+        route: '/tools/available-services',
+        advanced: true,
+        text: 'List all available services found in BlueOS serving http interfaces, and their'
+              + ' respective API documentations',
+      },
+      {
+        title: 'Bridges',
+        icon: 'mdi-bridge',
+        route: '/tools/bridges',
+        advanced: true,
+        text: 'Allows creating UDP/TCP to Serial bridges, used for communication to serial'
+              + ' devices from your topside computer',
+      },
+      {
+        title: 'File Browser',
+        icon: 'mdi-file-tree',
+        route: '/tools/file-browser',
+        advanced: true,
+        text: 'Browse all the files in BlueOS, useful for fetching logs,'
+              + ' tweaking configurations, and development',
+      },
+      {
+        title: 'NMEA Injector',
+        icon: 'mdi-map-marker',
+        route: '/tools/nmea-injector',
+        advanced: true,
+        text: 'Used for forwarding UDP NMEA stream into Ardupilot',
```

Comment:
```suggestion
        text: 'Used for forwarding UDP NMEA streams into Ardupilot',
```
Or should this be one? ("a UDP NMEA stream")